config/initializers/rules_of_origin.rb

```diff
@@ -0,0 +1,8 @@
+unless Rails.env.test?
+  Rails.application.reloader.to_prepare do
+    RulesOfOrigin::SchemeSet.current =
+      RulesOfOrigin::SchemeSet.new \
+        Rails.root.join \
+          "db/rules_of_origin/roo_schemes_#{TradeTariffBackend.service}.json"
```

Comment:
If you moved this to being a constant on the SchemeSet model you could avoid haven't to pass special snowflake args to the builder and just have a consistent api between all of the models.

Reply:
```ruby
RulesOfOrigin::SchemeSet.build
```